### ISSUE TITLE
reports of front-end slowdowns / freezes

### ISSUE THREAD

**Issue by domino14** [domino14/Webolith#351]
It could be fixed by upgrading the JS. Maybe it's because of an old computer. Should do some front-end flame graphs or something.

**Comment by dipscodehub** [comment]
@domino14 is this issue still open to contribute

**Comment by domino14** [comment]
yes - I haven't yet updated the frontend libraries; last time I tried it broke a few things like shuffling.
